### PR TITLE
HW: pages routers

### PR DESCRIPTION
**Pull request by centora**

### CODE REVIEW

File: src/components/loader/Loader.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import './loader.scss';
+
+export const Loader = ({ show }) => show && (
+  <div className="loader">&nbsp;</div>
```
Comment (iliatcymbal) [inline src/components/loader/Loader.js:4]: why need space here?

**Review by iliatcymbal — APPROVED** [review]
Looks nice

File: src/components/loader/Loader.js (hunk 1 above)
Comment (centora) [reply src/components/loader/Loader.js:4]: I thought it is not correct to leave empty div tag )

Comment (iliatcymbal) [reply src/components/loader/Loader.js:4]: no, just use `<div />`

**Pull request merged**